pandas/tseries/tests/test_frequencies.py

```diff
@@ -108,10 +113,6 @@ def test_to_offset_invalid(self):
         with tm.assertRaisesRegexp(ValueError, 'Invalid frequency: -2D:3H'):
             frequencies.to_offset('-2D:3H')
 
-        # ToDo: Must be fixed in #8419
-        with tm.assertRaisesRegexp(ValueError, 'Invalid frequency: .5S'):
-            frequencies.to_offset('.5S')
```

Comment:
Also pls add some tests whether this works with combined offset, like ``.5H.5S``.